src/Features/Core/Portable/SolutionCrawler/WorkCoordinator.IncrementalAnalyzerProcessor.cs

```diff
@@ -42,7 +42,11 @@ private partial class IncrementalAnalyzerProcessor
                 // NOTE: IDiagnosticAnalyzerService can be null in test environment.
                 private readonly Lazy<IDiagnosticAnalyzerService?> _lazyDiagnosticAnalyzerService;
 
-                private LogAggregator _logAggregator = new();
+                /// <summary>
+                /// The keys in this are either a string or a (string, Guid) tuple. See <see cref="SolutionCrawlerLogger.LogIncrementalAnalyzerProcessorStatistics"/>
+                /// for what is writing this out.
+                /// </summary>
+                private CountLogAggregator<object> _logAggregator = new();
```

Comment:
Refactoring this to be more strongly typed is out of scope for this PR. Until this PR, it wasn't even clear what the types were, so this is at least an improvement!